guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -29,7 +29,7 @@ You must update each virtual machine configuration so that they receive updates
 ** For {EL} 7:
 +
 ----
-# systemctl start rhsmcertd
+# systemctl enable --now rhsmcertd
```

Comment:
Previously this was only start, but not enable while for EL 6 it did both enable and start. I think this is fixing a bug.